docs/source/references/schema.json

```diff
@@ -3423,7 +3426,7 @@
       },
       {
         "name": "parameters",
-        "type": "Vec<Parameter>"
+        "type": "Option<Parameters>"
```

Comment:
Here are changes to the schema (https://github.com/hyperledger/iroha/issues/4813#issuecomment-2316873814)